gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -1019,10 +1031,18 @@ def main(args):  # noqa: D103
                 raise ValueError("Did not pass VCF field check")
 
             output_path = (
-                f"{qc_temp_prefix(data_type=data_type)}gnomad.{data_type}.test.chr{contig}.vcf.bgz"
+                f"{qc_temp_prefix(data_type=data_type)}gnomad.{data_type}.test{'.chr'+contig if contig else ''}.vcf.bgz"
                 if test
                 else release_vcf_path(contig=contig)
             )
+            # Match header order to info field order
+            logger.info("Matching header order to info field order...")
+            ordered_vcf_info_dict = {
+                f: header_dict["info"][f]
+                for f in list(ht.info)
+                if f in header_dict["info"]
+            }
+            header_dict.update({"info": ordered_vcf_info_dict})
 
             logger.info("Exporting VCF...")
             export_reference = build_vcf_export_reference("gnomAD_GRCh38")
```

Comment:
functionally doesn't matter, but could drop chrM since we never have it in these VCFS

Reply:
added this in https://github.com/broadinstitute/gnomad_methods/pull/640